packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter/services.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/gestures.dart' show DragStartBehavior;
 
+import '../foundation/change_notifier.dart';
```

Comment:
import should be :`package:flutter/foundation.dart`

Reply:
Done.